libs/designer-ui/src/lib/panel/recommendationpanel/operationSearchCard/index.tsx

```diff
@@ -39,6 +39,7 @@ export const OperationSearchCard = (props: OperationSearchCardProps) => {
       onClick={() => onCardClick()}
       style={style}
       data-automation-id={`msla-op-search-result-${convertUIElementNameToAutomationId(title)}`}
+      aria-label={`${title} ${description}`}
```

Comment:
If you're going to take this route, can we hide the inner info bubble from aria, otherwise voice assistants like Chromevox will end up reading it twice.

Reply:
I am seeing what I need to change to make that happen. Not sure if there is an equivalent of Chromevox for Windows where I can repro the output on Chromevox to understand how it outputs differently compared to nvda on Windows and how I can account for that

Reply:
The great thing about chromevox is that it's a browser extension: https://chrome.google.com/webstore/detail/screen-reader/kgejglhpjiefppelpmljglcjbhoiplfn

Totally usable cross platform :)

Reply:
Made change to conditionally hide InfoDot aria when we're providing custom aria-label